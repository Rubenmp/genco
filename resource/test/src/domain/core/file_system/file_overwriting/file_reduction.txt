The file overwriting process will remove <this part> file without trash at the end of the file.