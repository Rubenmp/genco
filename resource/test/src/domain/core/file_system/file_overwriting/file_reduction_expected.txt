The file overwriting process will remove <> from the file without trash at the end of the file.